osu.Game/Rulesets/Objects/Drawables/DrawableHitObject.cs

```diff
@@ -662,6 +662,15 @@ public virtual void OnKilled()
         /// </remarks>
         public virtual double MaximumJudgementOffset => HitObject.HitWindows?.WindowFor(HitResult.Miss) ?? 0;
 
+        /// <summary>
+        /// Whether the location of the hit should be snapped to the hit target before animating.
+        /// </summary>
+        /// <remarks>
+        /// This is how osu-stable worked, but notably is not how TnT works.
+        /// It results in less visual feedback on hit accuracy.
+        /// </remarks>
+        public bool SnapJudgementLocation { get; set; }
```

Comment:
@peppy Why is this all the way up here in `DrawableHitObject`? Surely it makes more sense to have it more localised, even to taiko alone?

I was scrolling past DHO today and I was quite surprised to see this here. The property doesn't even semantically make sense for any ruleset that isn't scrolling.

Reply:
This looks to have been a large oversight.